source/templates/act/_bylaw_help.html.haml

```diff
@@ -271,7 +271,7 @@
     .col-sm-6
       .single-sol
         %i.fa.fa-fw.fa-book
-        Familiarise yourself with the #{link_to 'Noise by-law', 'http://boris.local:4567/za-eth/act/by-law/2015/nuisances-behaviour-public-places/section/8/'} and #{link_to 'Building by-laws', 'http://www1.durban.gov.za/durban/government/bylaws/building-bylaws'}
+        Familiarise yourself with the #{link_to 'Noise by-law', '/za-eth/act/by-law/2015/nuisances-behaviour-public-places/section/8/'} and #{link_to 'Building by-laws', 'http://www1.durban.gov.za/durban/government/bylaws/building-bylaws'}
```

Comment:
The `/section/foo` pages don't exist any more, the content is all on one page. Use `/za-eth/act/by-law/2015/nuisances-behaviour-public-places/#section-8` instead